builtins.c

```diff
@@ -288,12 +288,14 @@ static value_t fl_vector_alloc(value_t *args, u_int32_t nargs)
         f = args[1];
     else
         f = FL_UNSPECIFIED;
+    fl_gc_handle(&f);
     v = alloc_vector((unsigned)i, f==FL_UNSPECIFIED);
     if (f != FL_UNSPECIFIED) {
```

Comment:
I wondered about whether the stack could move... but I guess not; your preference to use the stack tips the balance for me. I'll update my pull request.